app/graphql/albums/resolvers.js

```diff
@@ -0,0 +1,15 @@
+const config = require('../../../config'),
+  { url } = config.common.apiAlbums,
+  { getAlbumSources } = require('../../services/albums');
+
+const getAlbum = (_, params) => getAlbumSources(`${url}albums/${params.id}`);
+
+module.exports = {
+  Query: {
+    album: getAlbum
+  },
+  Album: {
+    artist: root => root.userId,
+    photos: root => getAlbumSources(`${url}photos?albumId=${root.id}`)
```

Comment:
This function name speaks about album sources, but we are getting photos here

Reply:
ok! I corrected it by naming it getAlbumAndPhotos, because this method is used for both depending on the url you send.